src/content/documentation/publish/version-compatibility.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: sidebar
+title: Firefox version compatibility
+permalink: /documentation/publish/version-compatibility/
+topic: Publish
+tags: [add-ons, intermediate, tutorial, webextensions, amo, distribution]
+contributors: [dotproto]
+last_updated_by: dotproto
+date: 2023-10-05
+---
+
+<!-- Page Hero Banner -->
+
+{% capture page_hero_banner_content %}
+
+# Firefox version compatibility
+
+Learn how to customize Firefox version compatibility settings on addons.mozilla.org and when to use this feature.
+
+{% endcapture %}
+{% include modules/page-hero.liquid,
+    content: page_hero_banner_content
+%}
+
+<!-- END: Overview Page Hero Banner -->
+
+<!-- Content with Table of Contents Module -->
+
+{% capture content_with_toc %}
+
+## `browser_specific_settings` vs. AMO compatibility settings
+
+There are two ways to control Firefox version compatibility settings for an add-on: using the `browser_specific_settings` field in the add-on's manifest or with the compatibility controls in AMO's version management interface. Each setting controls a different part of the addon
+
+Due to the way these two control mechanisms interact, it's possible for a developer to accidentally make an extension available for download in a browser that will fail to install it. To avoid this, we recommend that developes only use `browser_specific_settings` to control browser compatibility settings.
```

Comment:
```suggestion
Due to the way these two control mechanisms interact, it's possible for a developer to accidentally make an extension available for download in a browser that will fail to install it. To avoid this, we recommend that developers only use `browser_specific_settings` to control browser compatibility settings.
```